src/components-examples/material/list/list-single-selection/list-single-selection-example.html

```diff
@@ -1,9 +1,10 @@
-<mat-selection-list #shoes [multiple]="false">
-  @for (shoe of typesOfShoes; track shoe) {
-    <mat-list-option [value]="shoe">{{shoe}}</mat-list-option>
-  }
-</mat-selection-list>
-
-<p>
-  Option selected: {{shoes.selectedOptions.hasValue() ? shoes.selectedOptions.selected[0].value : 'None'}}
-</p>
+<form [formGroup]="form">
```

Comment:
Rather than replacing the example, I think we should add another one with reactive forms.